weekly-maintenance.sh

```diff
@@ -396,7 +396,7 @@ set -x
 jenkins-jobs/safe_git.sh sync_to_origin "git@github.com:Khan/webapp" "automated-commits"
 jenkins-jobs/safe_git.sh sync_to_origin "git@github.com:Khan/network-config" "master"
 
-( cd webapp && make deps )
+( cd webapp && make deps python_deps )
```

Comment:
So I understand, right now this is actually redundant as `make deps` still runs the python_deps, but in the near future that will not be the case, correct?

Either way, LGTM, just want to make sure I understand the current and desired future states.

Reply:
Yes, that's correct!